pkg/storage/store.go

```diff
@@ -125,6 +129,10 @@ type store struct {
 
 // NewStore creates a new Loki Store using configuration supplied.
 func NewStore(cfg Config, schemaCfg SchemaConfig, chunkStore chunk.Store, registerer prometheus.Registerer) (Store, error) {
+	index := ActivePeriodConfig(schemaCfg.Configs)
+	indexTypeStats.Set(schemaCfg.Configs[index].IndexType)
+	objectTypeStats.Set(schemaCfg.Configs[index].ObjectType)
+	schemaStats.Set(schemaCfg.Configs[index].Schema)
```

Comment:
We should add a check here to ensure this config exists to make this more safe

Reply:
done